docs/libblockdev-sections.txt

```diff
@@ -156,6 +156,7 @@ bd_utils_exec_and_report_error_no_progress
 bd_utils_exec_and_report_progress
 bd_utils_init_logging
 bd_utils_init_prog_reporting
+bd_utils_init_prog_reporting_thread
```

Comment:
Should we also add a function to unset the progress reporting handler for a thread?

Reply:
Yes, but I think it would be a nice convenience function.

Reply:
Ok, I have added it by using static empty function which allows to detect muting in `bd_utils_prog_reporting_initialized`. Checking in `bd_utils_report_progress` is not necessary because the code will usually test if reporting is initialized and otherwise not call this function, but anyway it would only make an empty call which is not much overhead.